persistence-jdbc/core/src/main/resources/reference.conf

```diff
@@ -38,6 +38,9 @@ db.default {
 
     # 5 * number of threads
     maxConnections = 100
+
+    # if true, a Mbean for AsyncExecutor will be registered
+    registerMbeans = false
```

Comment:
Decide to add it to this PR. This can be useful when debugging slick. 

`AsyncExecutor` provides this option, but we were not using it.